### ISSUE TITLE
no normalize_embd_dim and cb_only in model.py

### ISSUE THREAD

**Issue by shenyang0111ucf** [OSU-NLP-Group/Text2SQL-Error-Detection#1]
Hi,
I found no normalize_embd_dim and cb_only parameter in model.py. Could you tell me if they are useful for parts of the paper (ablation study?)

Regards,
Shenyang Liu

**Comment by ChenSJ98** [comment]
Hi Shenyang,

Sorry for the late reply. These are arguments were used when we were developing the model. They're not related to the results in the paper. Thanks for pointing this out, we will update the training script accordingly.
